### PR TITLE
Find_eclipse_candidates

### PR DESCRIPTION
**Pull request by jrka**

### CODE REVIEW

**Comment by kelle** [comment]
This looks great!!  Only thing I see is that Solutions should be separated out into dedicated notebook. Other than that, I think this is ready for review by someone at MAST.

File: notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb [hunk 1]
```diff
@@ -0,0 +1,754 @@
+{
```
Comment (hmlewis-astro) [inline notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb:1]: <ul><li>On your second bullet point ("Narrow down a list..."), I'd suggest rewording slightly: "Find the binary candidates most likely to be observed as eclipsing binaries by using a set of parameters from the APOGEE catalog."</li><li>On your third bullet point ("Determine if a stellar system..."), slight rewording: "Determine if a candidate system in the APOGEE catalog has a light curve in (a) the TESS Data for Asteroseismology Lightcurves archive from TASOC (the TESS Asteroseismic Science Operations Center) or (b) the TESS Lightcurves From The MIT Quick-Look Pipeline ("QLP") archive using&nbsp;<code>astroquery</code>&nbsp;and the MAST archive." -- Note, spelling correction of "Asteroseismic".</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (hmlewis-astro) [inline notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb:1]: <p>Some very small changes in this section!</p><p> </p><ul><li>Remove comma after "The <a href="https://www.sdss.org/surveys/apogee-2/" target="_blank">Apache Point Observatory Galactic Evolution Experiment 2 (APOGEE-2)</a>" </li><li>Second paragraph, second sentence, slight rewording: "This survey was used to search for stars with spectral lines showing the tell-tale radial velocity shifts associated with stellar binaries."</li><li>Second paragraph, third sentence, changing "binary stars" to "companion stars", to make it more general: "...eclipses of primary stars by companion stars would also be observed by TESS."</li><li>Third paragraph, spelling of "Asteroseismic" (missing the first "i"; "Asteroseismology" is correct though!)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (hmlewis-astro) [inline notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb:1]: <p>Correct spelling of "companions" in the second sentence</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (hmlewis-astro) [inline notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #7.    # JRK, I get an annoying units error from the table. Pretty sure it's Vizier's fault? 
</pre><p>I agree that this is either an issue with Vizier or the table itself. I think the problem is with [Fe/H] and [a/Fe] parameters which are given in solar units (written as "[Sun]" in the Vizier table). You might be able to silence Vizier warnings?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (hmlewis-astro) [inline notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb:1]: <ul><li>Second paragraph, first sentence, spelling correction: "...we'll start our search with the most likely candidates"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (hmlewis-astro) [inline notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb:1]: <p>Small suggestion on the second set of bullet points:</p><ul><li>Second point, suggestion rewording: "...qmin, the mass ratio of the companion to the primary star,"</li></ul><p> </p><p>At the end of this section, you might also want to just put a note along the lines of "For details on the derivation of these parameters, please refer to the Price-Whelan et al. 2018 paper."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (hmlewis-astro) [inline notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #9.    print(len(tbl))
</pre><p>I would also have this print out what "17" means, e.g., <code>print("Number of systems meeting our selection criteria:\n", len(tbl))</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (hmlewis-astro) [inline notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb:1]: <p>Just noticed it here (probably because it's bold and in the title), but I think "light curve" is generally 2 words. I know that isn't the case for the TASOC and QLP titles though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (hmlewis-astro) [inline notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #5.    print(obs_tbl)
</pre><p>Again, I'd have it print out what "8" means, e.g. "Number of Obs in MAST:"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (hmlewis-astro) [inline notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #7.    # JRK another error here, can change from using a coordinate string to astropy coords
</pre><p>Would probably be good to convert to astropy coordinates.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (hmlewis-astro) [inline notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb:1]: <p>Index 19? Do you mean index 1 or 11? Likely 1 based on the next cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (hmlewis-astro) [inline notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb:1]: <p>For now, I would just say "...we will example the light curves individually." because it's not clear yet what "all together" means, and could be a point of confusion.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (hmlewis-astro) [inline notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb:1]: <p>These are nice exercises!</p><p> </p><p>In item 1, might also suggest sorting by primary mass (M1) or the velocity semi-amplitude (K).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Comment (hmlewis-astro) [inline notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #11.    # but many of the periods are extremely long; too long to be observed by TESS.
</pre><p>I'd say "unlikely to be observed by TESS". There's a <em>chance</em> that TESS <em>happened</em> to catch them at the right moment, it's just not very likely given the long period.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (jrka) [reply notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb:1]: <p>Just reran this on a different computer, and I don't get the error. I'll have to investigate if there's a version difference responsible for it.</p>

Comment (jrka) [reply notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb:1]: <p>I think it's important to keep "minimum," but I did make the change from "or" to "the." </p>

Comment (jrka) [reply notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb:1]: <p>Yep, definitely meant 1. I realized that the "2 of our 17" could possibly change in the future? Are more processed light curves being added to QLP and TASOC? I added "(as of the writing of this tutorial)" to the end of that sentence. However, I don't <em>think</em> my choice of index 1 would change in the future becuase it's based on the Vizier table  and the order shouldn't even change. Should I hard code my choice of RA and Dec instead?</p>

File: notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb [hunk 2]
```diff
@@ -0,0 +1,721 @@
...
```
Comment (jrka) [inline notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #8.    tbl=tbl[~tbl['qmin'].mask] # Is this gross? 
</pre><p>Remove the comment if no one has objections :-)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

File: notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb (hunk 1 above)
Comment (hmlewis-astro) [reply notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb:1]: <p>Yes, QLP and TASOC both deliver new data frequently, so there's a chance that there will be more targets with light curves in the future. I think adding "as of the writing of this tutorial" is a good fix for the first sentence, but I would specify the RA/Dec rather than the index number.</p>

Comment (jrka) [reply notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb:1]: <p>For what it's worth, I get the warning on my computer that is most updated. I removed the comment. The warning doesn't affect the workflow. </p>
